docs/get-started/query-analytics.md

```diff
@@ -18,7 +18,7 @@ The dashboard contains three panels:
 - the [Details Panel](#details-panel).
 
 !!! note alert alert-primary ""
-    Query Analytics data retrieval is not instantaneous and can be delayed due to network conditions. In such situations QAN data continues to be gathered and is delivered as soon as the connection is re-established.
+    Query Analytics data retrieval is not instantaneous and can be delayed due to network conditions. In such situations *no data* is reported and a gap appears in the sparkline.
```

Comment:
small amount of data will be gathered(up to 1 hour) and buffered in memory and reported when connection is back.

Reply:
@BupycHuk This has been removed from PMM 2.41.0. Please reaffirm.

Reply:
The message I shared is a current state. we still gather some data and buffer it, so they will survive short network issues. We wanted to extend data buffering for longer period, but had to revert that.

Reply:
Fixed.